python/cuml/linear_model/elastic_net.pyx

```diff
@@ -222,6 +222,18 @@ class ElasticNet(Base,
 
         return self
 
+    def set_params(self, **params):
+        super().set_params(**params)
+        self._check_alpha(self.alpha)
+        self._check_l1_ratio(self.l1_ratio)
+        shuffle = self.selection == 'random'
+        self.solver_model = CD(fit_intercept=self.fit_intercept,
+                               normalize=self.normalize, alpha=self.alpha,
+                               l1_ratio=self.l1_ratio, shuffle=shuffle,
+                               max_iter=self.max_iter, handle=self.handle,
+                               output_type=self.output_type)
```

Comment:
Why not just call solver.set_params as with the mbsgd ones?

Reply:
Some parameters are different between the class and it's solver. But I updated them to adjust the parameters.